cypress/support/api_commands.js

```diff
@@ -177,6 +177,44 @@ Cypress.Commands.add('apiAddUserToTeam', (teamId, userId) => {
     });
 });
 
+/**
+ * Join teammates directly via API
+ * This API assume that the user is logged in and has cookie to access
+ * @param {String} name - The name arg previously used during team-create (unique but not GUID)
+ * @param {Integer} size - The amount of users to join
+ * All parameter required
+ */
+Cypress.Commands.add('apiBatchJoinTeam', (name, size = 60) => {
+    return cy.getCookie('MMUSERID').then((cookie) => {
+        const admin = cookie.value;
+        cy.request({method: 'GET', url: `/api/v4/teams/name/${name}`}).then((teamResp) => {
+            const key = teamResp.body.id;
+            cy.request({method: 'GET', url: `/api/v4/users?per_page=${size}`}).then((userResp) => {
+                // Shape the users list into batch request body, skip the team admin.
+                const ulist = userResp.body.filter((u) => u.id !== admin).map((user) => {
+                    return {team_id: key, user_id: user.id, roles: 'system_user', delete_at: 0, scheme_user: true, scheme_admin: true, explicit_roles: 'system_user'};
+                });
+
+                // The API errors when the list is more than 20 users, so do chunks.
+                for (let i = ulist.length; i > 20; i -= 20) {
```

Comment:
```suggestion
const chunkedUsers = Cypress._.chunk(ulist, 20);
                chunkedUsers.forEach((list) => {
                    // Send request on twenty
                    cy.request({method: 'POST', headers: {'X-Requested-With': 'XMLHttpRequest'}, url: `/api/v4/teams/${key}/members/batch`, body: list, qs: {team_id: key}});
                });
```

Leverage lodash to chunk the users so you have an array of arrays to loop over.

Reply:
Yes, I replaced the loop with Cypress._.chunk in today's PR.